.github/workflows/mise.yml

```
@@ -0,0 +1,36 @@
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 100
+          submodules: recursive
+      - name: Install Mise
+        uses: jdx/mise-action@v2
+        with:
+          version: 2024.12.14
+          install: true
+          cache: true
+          experimental: true 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)